spec/helpers/application_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+describe ApplicationHelper do
+  describe '#tooltip' do
+    it 'creates a div containing aria label with text and image' do
+      tooltip_text = 'foobar'
```

Comment:
I can't remember where I picked it up from but I've certainly heard it from several sources and tend to agree.

here is some discussion google found on it:
http://stackoverflow.com/questions/6453235/what-does-damp-not-dry-mean-when-talking-about-unit-tests

I'm not a super proponent of this POV, but in cases like this I think the use of a variable makes the test a little less readable.